python/ray/data/read_api.py

```diff
@@ -370,6 +372,7 @@ def read_datasource(
             min_safe_parallelism,
             inmemory_size,
             read_tasks,
+            reader,
         ) = ray.get(
             get_read_tasks.remote(
```

Comment:
how is `get_read_tasks` being postponed if it's still being called here?

Reply:
Added a comment below, this is needed for `LazyBlockList` code path.